bftengine/src/bftengine/ReplicasInfo.cpp

```diff
@@ -144,6 +144,10 @@ ReplicasInfo::ReplicasInfo(const ReplicaConfig& config,
         if (start != end) LOG_INFO(GL, "Principal ids in _idsOfInternalClients: " << start << " to " << end - 1);
         return ret;
       }()} {
+  _operator_id = config.operatorEnabled_
+                     ? static_cast<PrincipalId>(numberOfReplicas() + _idsOfClientProxies.size() +
+                                                _idsOfExternalClients.size() + _idsOfClientServices.size())
```

Comment:
Do we have some centralized place for this calculation? I assume we calculate it in different places. and it might change sometime along the way...

Reply:
The calculation is listed here https://github.com/vmware/concord-bft/blob/master/bftengine/src/bftengine/ReplicasInfo.cpp#L19

Reply:
If it is changed, then we need to change the whole calculation of in this component